src/main/java/org/dependencytrack/tasks/vulnerabilitypolicy/AbstractVulnerabilityPolicyFetcher.java

```diff
@@ -1,174 +1,36 @@
 package org.dependencytrack.tasks.vulnerabilitypolicy;
 
 import alpine.Config;
-import alpine.common.logging.Logger;
-import alpine.server.util.DbUtil;
-import com.fasterxml.jackson.databind.JsonNode;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
-import com.fasterxml.jackson.datatype.jsonorg.JsonOrgModule;
-import com.networknt.schema.JsonSchema;
-import com.networknt.schema.JsonSchemaFactory;
-import com.networknt.schema.SpecVersion;
-import com.networknt.schema.ValidationMessage;
-import org.apache.http.HttpEntity;
 import org.apache.http.client.methods.CloseableHttpResponse;
 import org.apache.http.client.methods.HttpGet;
 import org.apache.http.client.methods.HttpHead;
 import org.apache.http.client.methods.HttpUriRequest;
 import org.apache.http.impl.client.CloseableHttpClient;
 import org.dependencytrack.common.ConfigKey;
 import org.dependencytrack.common.HttpClientPool;
-import org.dependencytrack.model.VulnerabilityPolicy;
-import org.dependencytrack.persistence.QueryManager;
-import org.dependencytrack.policy.cel.CelPolicyScriptHost;
-import org.dependencytrack.policy.cel.CelPolicyType;
+import org.dependencytrack.tasks.vulnerabilitypolicy.blobstorage.BlobStorageAccessHandler;
 import org.dependencytrack.util.HttpUtil;
-import org.projectnessie.cel.tools.ScriptCreateException;
 
-import java.io.BufferedReader;
 import java.io.IOException;
-import java.io.InputStreamReader;
-import java.nio.charset.StandardCharsets;
-import java.sql.Connection;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Set;
-import java.util.zip.ZipEntry;
-import java.util.zip.ZipInputStream;
-
-import static org.apache.commons.io.IOUtils.resourceToString;
 
 public abstract class AbstractVulnerabilityPolicyFetcher implements BlobStorageAccessHandler {
```

Comment:
This class appears to not fulfill any use anymore. The functionality it has seems to be exclusive to the `NginxStorageHandler`, as `S3StorageHandler` doesn't use / need any of it. Perhaps the remaining code in here can be moved to `NginxStorageHandler`, and this class can be removed?